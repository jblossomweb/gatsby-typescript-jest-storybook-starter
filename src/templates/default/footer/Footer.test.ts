import snapshotTests from "../../../../utils/story-snapshot-tests"
import { stories } from "./Footer.stories"

describe("templates/default/footer", () => {
  snapshotTests(stories)
})
