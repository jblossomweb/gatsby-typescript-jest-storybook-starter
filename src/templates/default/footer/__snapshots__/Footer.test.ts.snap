// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`templates/default/footer default matches snapshot 1`] = `
<footer>
  Â© 
  2020
  , Built with
   
  <a
    href="https://www.gatsbyjs.org"
    target="_blank"
  >
    Gatsby
  </a>
</footer>
`;
